server/utils/utils.ts

```diff
@@ -28,3 +28,17 @@ export const initialiseName = (fullName: string | undefined | null): string | nu
   }
   return `${firstName[0]}. ${array.reverse()[0]}`
 }
+
+export const escapeHTML = (str: string | undefined): string | undefined =>
```

Comment:
Would definitely prefer we don't write our own escape functions. See comment above. The formatting and escaping should be done in the njk view